lib/migrations/migration.ts

```diff
@@ -10,6 +11,7 @@ export class MigrationManager {
       "1.4.0": v1_4_0,
       "2.0.4": v2_0_4,
       "2.3.0": v2_3_0,
+      "2.5.0": v2_5_0,
```

Comment:
As we already move to 2.5.0 version. It would be better to have another PR `2.5.1` or `2.6.0` then merge 2.4.0 and 2.5.0 migration to the new version.